wgpu/src/lib.rs

```diff
@@ -1465,6 +1466,12 @@ pub struct VertexState<'a> {
     /// The name of the entry point in the compiled shader. There must be a function with this name
     /// in the shader.
     pub entry_point: &'a str,
+    /// Specifies the values of pipeline-overridable constants in the shader module module.
```

Comment:
question: `module module` seems wrong; should this be just `module`?

This also happens in a few other places.